### ISSUE TITLE
Case sensitive file systems make trouble

### ISSUE THREAD

**Issue by wolfv** [conda/rattler#460]
The `ncurses` package contains two files that share the same name in different capitalizations (at least on Linux).

Users using docker on macOS have a problem (https://github.com/prefix-dev/pixi/discussions/594#discussioncomment-7951695) because linking fails.

I think we should fix this in rattler so that one of the two files "wins" and the other one is ignored.

**Comment by pavelzw** [comment]
related https://github.com/conda-forge/ncurses-feedstock/issues/73

**Comment by baszalmstra** [comment]
Now that we have awesome clobbering support, is this still an issue?

**Comment by pavelzw** [comment]
yes, i tried out installing `ncurses==6.1` in a docker container that's mounted on macOS and pixi doesn't complain anymore
